### DISCUSSION TITLE
Does the device emulate writing of CD-R and CD-RW ?

### DISCUSSION THREAD

**Discussion by edmaher** [ZuluIDE/ZuluIDE-firmware#146] [Q&A] [answered]
Some machines such as the Korg D32XD & D16XD Digital multitrack  that have an internal CD-ROM/ CD-RW writer.
CD-ROM and CD-RW media is increasingly rare, so the capability to replace the drive with a ZuluIDE unit would seem to be a good solution.
I think in this specific case, the USB connection may work for some functions, but it seems like there would be other and similar use-cases.

**Comment by aperezbios** [comment, marked as answer]
@edmaher  it does not at this time. It may be possible to implement that sort of functionality, but it currently isn't on our roadmap.
